Caboodle/Sms/Sms.netstandard.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Sms
+    {
+        public static bool IsComposeSupported
```

Comment:
I think we still need to decide if a bool and this naming is what we want to end up with.

Reply:
Yeah. We may want to have some support level thing: Unsupported, Supported, SupportedWithBackgroundSend or something along those lines. This is more a feature that works when there is a SIM card of some sort.

Reply:
I think that probably each class should be small enough that a single Enum should represent what it needs...

So we could introduce "SmsBackground" for instance and that is a new API with a new Enum. I think that is almost better to be honest with you.